tools/code_format/check_format.py

```diff
@@ -321,6 +321,14 @@ def checkTools(self):
               CLANG_FORMAT_PATH))
 
     def checkBazelTool(name, path, var):
+      if not path:
+        error_messages.append(
+            "`buildifier` or `buildozer` not found, " +
+            "Make sure you have installed them. See step 3 and 4 on: " +
+            "https://github.com/envoyproxy/envoy/blob/master/bazel/README.md#quick-start-bazel-build-for-developers for details."
+        )
+        return error_messages
```

Comment:
This is less descriptive than what the script already says in below, what do you want to achieve with this?

```
        error_messages.append("Command {} not found. If you have {} installed, but the binary "
                              "name is different or it's not available in $GOPATH/bin, please use "
                              "{} environment variable to specify the path. Example:\n"
                              "    export {}=`which {}`\n"
                              "If you don't have {} installed, you can install it by:\n"
                              "    go get -u github.com/bazelbuild/buildtools/{}".format(
                                  path, name, var, var, name, name, name)
```

Reply:
@lizan 
Hi, when the `path` is NoneType, [line 332](https://github.com/envoyproxy/envoy/pull/13614/files#diff-d62e3cec02d574d838802c939ec86a175a3672641a6e67d36d80b3dda91fd6a5R332
) will throw an exception and exit, and will not go to code below. 
This change is to avoid this exit.

Reply:
ping @lizan